index.rst

```diff
@@ -311,6 +311,29 @@ For a short period, just using ``if`` guards to allow both approaches to coexist
 
 Another approach that may be better for continued support of both approaches would be to implement a new ``LimitedButler`` subclass to be backed by a prepopulated registry, which should reduce or even eliminate the need for other middleware code (specifically ``SingleQuantumExecutor``) to be aware of the mode in which execution is occurring.
 
+Performance
+^^^^^^^^^^^
+
+To compare overall performance of the execution and the quantum-backed butler we selected Step 1 on DC2 test-med-1.
+For each run, the workflow consisted for of total 57,305 quanta distributed evenly between five tasks: ``isr``, ``characterizeImage``, ``calibrate``, ``writeSourceTable``, ``transformSourceTable``.
+LSST Batch Processing Service (BPS) added two more jobs to the workflow: ``pipetaskInit``, ``finalJob``.
```

Comment:
```suggestion
Rubin's BPS added two more jobs to the workflow: ``pipetaskInit``, ``finalJob``.
```

I think we've decided that BPS isn't a "service" and more like a "system" but I don't think we've made a formal statement to that effect and have instead not defined BPS at all.